### ISSUE TITLE
Onboard DEX to Universal Pipeline PRODUCTION

### ISSUE THREAD

**Issue by arnejduranovic** [CDCgov/prime-reportstream#9244]
### User Story:
As a sender to DEX, I, ReportStream, need to help onboard DEX as a receiver to production so I can send them production data.

### Description/Use Case
DEX should be set up to receive data from RS on production.

### Dev Notes:
This is a follow-on to #9243. Once that story is complete and we are ready to start sending data in production, we can repeat the steps as part of this story.

### Acceptance Criteria

- [ ]  Worked with Boris (tpz7@cdc.gov) from DEX to confirm RS can send data to DEX in the way DEX wants it in production

**Comment by chris-kuryak** [comment]
DEX is being deccomissioned. No longer needed.
